src/test.py

```diff
@@ -0,0 +1,48 @@
+from threading import Thread
+import time
+
+global cycle
+cycle = 0.0
+
+class Hello5Program:  
+    def __init__(self):
+        self._running = True
+
+    def terminate(self):  
+        self._running = False  
+
+    def run(self):
+        global cycle
+        while self._running:
+            time.sleep(5) #Five second delay
+            cycle = cycle + 1.0
+            print ('5 Second Thread cycle+1.0 - ')
+
+class Hello2Program:  
+    def __init__(self):
+        self._running = True
+
+    def terminate(self):  
+        self._running = False  
+
+    def run(self):
+        global cycle
+        while self._running:
+            time.sleep(2) #Five second delay
+            cycle = cycle + 0.5
+            print ('5 Second Thread cycle+1.0 - ')
+
+#Create Class
```

Comment:
Cool! I like this threading approach! Is there a way to modularize the delay? For example, so that the user can control how often the system is polling the sensors? How quickly do you guys think the pH control system needs to update/operate for good response?

Reply:
I believe our client mentioned this in meeting, and the pH controlling system should operate every 24hrs. The delays here are used for testing right now